src/main/java/it/pagopa/pn/pdfraster/configuration/aws/PdfTransformationConfiguration.java

```diff
@@ -47,9 +52,32 @@ public PdfTransformationConfigParams pdfTransformationConfigurationFromParameter
 
         String parameterValue = response.parameter().value();
 
-        return  jsonUtils.convertJsonStringToObject(parameterValue, PdfTransformationConfigParams.class);
+        return jsonUtils.convertJsonStringToObject(parameterValue, PdfTransformationConfigParams.class);
 
     }
 
+    private List<TransformationEnum> parseTransformations(String transformations) {
```

Comment:
chiarirei, dopo opportuno test, l'effettivo comportamento in merito all'ordine tra portrait e crop/scale, evidenziando in un commento nel codice il comportamento e se effettivamente, come sembra, non sia possibile fare una scalatura/crop prima di una rotazione

Reply:
Il test ha evidenziato i seguenti risultati:

- scale;portrait: nessuna differenza evidente con portrait;scale
- crop;portrait: equivalente a crop